tests/appsec/test_ip_blocking_full_denylist.py

```diff
@@ -56,16 +55,7 @@ def validate(data):
 
     def setup_blocked_ips(self):
         NOT_BLOCKED_IP = "42.42.42.3"
-        BLOCKED_IPS = [
-            "245.159.180.146",
-            "126.29.6.251",
-            "59.9.47.235",
-            "179.140.243.163",
-            "136.26.224.167",
-            "165.160.156.36",
-            "172.2.59.155",
-            "169.49.236.135",
-        ]
+        BLOCKED_IPS = [f"12.8.{a}.{b}" for a in range(10) for b in range(125)]
```

Comment:
Could you just add a comment explaining this list, the reason for `10` and `250`, and where to look for its equivalent in remote json files / how to modify them ?